modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,7 +10,7 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate the link:https://github.com/spadgett/console-plugin-template/blob/main/template.yaml[template] by running the following command:
```

Comment:
so if we remove the link to `template.yaml` how user can get the `template.yaml` file?